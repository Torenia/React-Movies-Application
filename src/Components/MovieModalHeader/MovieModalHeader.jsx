import styled from 'styled-components';

export const MovieModalHeader = styled.div`
  background-color: #232323;
  position: relative;
    
  h1 {
    color: #fff;
    text-transform: uppercase;
    font-weight: normal;
    padding: 30px 0 10px 80px;
    margin: 0;
  }
    
  span {
    position: absolute;
    top: 15px;
    right: 0;
    font-size: 15px;
    background: #232323;
    width: 35px;
    height: 35px;
    border: none;
    border-radius: 50%;
    outline:none;
    cursor: pointer;
  
    &:focus {
     outline:none;
    }
  
    &:after {
     content: "\\2716";
     display: inline-block;
     color: #fff;
    }
  }
`;
