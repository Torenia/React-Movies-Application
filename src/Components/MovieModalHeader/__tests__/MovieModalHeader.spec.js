import React from 'react';

import { MovieModalHeader } from '../MovieModalHeader';

describe('<MovieModalHeader />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieModalHeader />);
  });

  it('should render a movie details header', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
