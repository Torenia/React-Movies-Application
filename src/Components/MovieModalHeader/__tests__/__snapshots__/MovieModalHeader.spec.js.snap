// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieModalHeader /> should render a movie details header 1`] = `
.c0 {
  background-color: #232323;
  position: relative;
}

.c0 h1 {
  color: #fff;
  text-transform: uppercase;
  font-weight: normal;
  padding: 30px 0 10px 80px;
  margin: 0;
}

.c0 span {
  position: absolute;
  top: 15px;
  right: 0;
  font-size: 15px;
  background: #232323;
  width: 35px;
  height: 35px;
  border: none;
  border-radius: 50%;
  outline: none;
  cursor: pointer;
}

.c0 span:focus {
  outline: none;
}

.c0 span:after {
  content: "\\2716";
  display: inline-block;
  color: #fff;
}

<div
  className="c0"
/>
`;
